modules/my_health/spec/request/v1/triage_teams_request_spec.rb

```diff
@@ -8,64 +8,109 @@
+  context 'when sm session policy is enabled' do
+    before do
+      Flipper.enable(:mhv_sm_session_policy)
+      Timecop.freeze(Time.zone.parse('2017-05-01T19:25:00Z'))
+    end
 
-    before { get '/my_health/v1/messaging/recipients' }
+    after do
+      Flipper.disable(:mhv_sm_session_policy)
+      Timecop.return
+    end
 
-    include_examples 'for user account level', message: 'You do not have access to messaging'
-    include_examples 'for non va patient user', authorized: false, message: 'You do not have access to messaging'
-  end
+    context 'when NOT authorized' do
+      before do
+        VCR.insert_cassette('sm_client/session_error')
+        get '/my_health/v1/messaging/recipients'
+      end
 
-  context 'Advanced User' do
-    let(:mhv_account_type) { 'Advanced' }
+      after do
+        VCR.eject_cassette
+      end
 
-    before { get '/my_health/v1/messaging/recipients' }
+      include_examples 'for user account level', message: 'You do not have access to messaging'
+    end
 
-    include_examples 'for user account level', message: 'You do not have access to messaging'
-    include_examples 'for non va patient user', authorized: false, message: 'You do not have access to messaging'
-  end
+    context 'when authorized' do
+      before do
+        VCR.insert_cassette('sm_client/session')
+      end
 
-  context 'Premium User' do
-    let(:mhv_account_type) { 'Premium' }
+      after do
+        VCR.eject_cassette
+      end
 
-    context 'not a va patient' do
-      before { get '/my_health/v1/messaging/recipients' }
+      it 'responds to GET #index' do
+        VCR.use_cassette('sm_client/triage_teams/gets_a_collection_of_triage_team_recipients') do
+          get '/my_health/v1/messaging/recipients'
+        end
 
-      let(:va_patient) { false }
-      let(:current_user) do
-        build(:user, :mhv, :no_vha_facilities, va_patient:, mhv_account_type:)
+        expect(response).to be_successful
+        expect(response.body).to be_a(String)
+        expect(response).to match_response_schema('triage_teams')
       end
 
-      include_examples 'for non va patient user', authorized: false, message: 'You do not have access to messaging'
-    end
+      it 'responds to GET #index when camel-inflected' do
+        VCR.use_cassette('sm_client/triage_teams/gets_a_collection_of_triage_team_recipients') do
+          get '/my_health/v1/messaging/recipients', headers: { 'X-Key-Inflection' => 'camel' }
+        end
 
-    it 'responds to GET #index' do
-      VCR.use_cassette('sm_client/triage_teams/gets_a_collection_of_triage_team_recipients') do
-        get '/my_health/v1/messaging/recipients'
+        expect(response).to be_successful
+        expect(response.body).to be_a(String)
+        expect(response).to match_camelized_response_schema('triage_teams')
       end
+    end
+  end
+
+  context 'when legacy sm policy' do
+    before do
+      Flipper.disable(:mhv_sm_session_policy)
```

Comment:
same here, is you allow(Flipper)... you won't have the possibility of running into flaky specs